<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDialog_Animations</class>
 <widget class="QDialog" name="CDialog_Animations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Последовательность анимации тайлов</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,0,0">
     <item row="0" column="0" rowspan="3" colspan="2">
      <widget class="QScrollArea" name="cScrollArea_Tiles">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="cScrollAreaWidgetContents_Tiles">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>359</width>
          <height>280</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="cPushButton_Delete">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="cPushButton_Clear">
       <property name="text">
        <string>Удалить все</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="3">
      <widget class="QScrollArea" name="cScrollArea_TilesSequence">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="cScrollAreaWidgetContents_TilesSequence">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>146</width>
          <height>218</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="2" colspan="3">
      <widget class="QPushButton" name="cPushButton_Add">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="3">
      <widget class="QPushButton" name="cPushButton_Apply">
       <property name="text">
        <string>Применить</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="cPushButton_Cancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="2" colspan="3">
      <widget class="QComboBox" name="cComboBox_AnimationMode"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
